src/layer.coffee

```diff
@@ -111,6 +111,10 @@ class Layer
     position = Point(rows, columns)
     @fromPositionInLayer(@toPositionInLayer(position, @source, clip: direction), @source)
 
+  startsWith: (string, start=Point(0, 0)) ->
+    position = @positionOf(string, start)
```

Comment:
I actually think you should implement this as its own method all the way to the raw characters layer to avoid scanning past the current position. There's a `String::startsWith` method available in ES6 you can use for the implementation there.